packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header-title.component.tsx

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import styles from './vital-header-title.component.scss';
+import styles from './vitals-header-title.component.scss';
 import WarningFilled20 from '@carbon/icons-react/es/warning--filled/20';
 import ChevronDown16 from '@carbon/icons-react/es/chevron--down/16';
 import ChevronUp16 from '@carbon/icons-react/es/chevron--up/16';
 import dayjs from 'dayjs';
 import { Button } from 'carbon-components-react';
 import { useTranslation } from 'react-i18next';
 import { attach } from '@openmrs/esm-framework';
-import { PatientVitals } from '../vitals-biometrics.resource';
+import { PatientVitals } from '../vitals.resource';
 import { patientVitalsBiometricsFormWorkspace } from '../../constants';
 
-interface VitalsHeaderStateTitleProps {
+interface VitalsHeaderTitleProps {
   view: string;
   vitals: PatientVitals;
   toggleView(): void;
   showDetails: boolean;
   showRecordVitals: boolean;
 }
 
-const VitalsHeaderStateTitle: React.FC<VitalsHeaderStateTitleProps> = ({
+const VitalsHeaderTitle: React.FC<VitalsHeaderTitleProps> = ({
```

Comment:
This is still a funny name. Is it a header, or a title? Or a header tile? Or the title of the header, because for some reason the body has its own title, which is different??